parsl/executors/taskvine/stub_staging_provider.py

```diff
@@ -0,0 +1,37 @@
+import logging
+from concurrent.futures import Future
+from typing import Optional
+
+from parsl.app.futures import DataFuture
+from parsl.data_provider.files import File
+from parsl.data_provider.staging import Staging
+from parsl.utils import RepresentationMixin
+
+logger = logging.getLogger(__name__)
+
+known_url_schemes = ["http", "https", "taskvinetemp"]
+
+
+class StubStaging(Staging, RepresentationMixin):
+
+    def can_stage_in(self, file):
+        logger.debug("Task vine staging provider checking passthrough for {}".format(repr(file)))
+        return file.scheme in known_url_schemes
+
+    def can_stage_out(self, file):
+        logger.debug("Task vine staging provider checking passthrough for {}".format(repr(file)))
+        return file.scheme in known_url_schemes
+
+    def stage_in(self, dm, executor: str, file: File, parent_fut: Optional[Future]) -> Optional[DataFuture]:
+        if file.netloc == '':
```

Comment:
i don't really understand the logic of these lines - theres a lot of moving fields around in weird ways in those two lines. for example using `netloc` for a filename is unusual. what's wrong with using `local_path` there?